Easydict/NewApp/View/MenuItemView.swift

```diff
@@ -24,6 +25,7 @@ final class MenuItemStore: ObservableObject {
 @available(macOS 13, *)
 struct MenuItemView: View {
     @ObservedObject private var store = MenuItemStore()
+    @Environment(\.openSettings) private var openSettings
```

Comment:
It seems that this variable is not used?

Reply:
Yes, forgot to remove it.  Another workaround used this at first.